services/node/node-lts.service.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const NodeVersionBase = require('./node-base')
+const { versionColorForRangeLts } = require('./node-version-color')
+
+module.exports = class NodeLtsVersion extends NodeVersionBase {
+  static get path() {
+    return 'v-lts'
+  }
+
```

Comment:
Should probably have different default badge labels between these two services, `node lts` for this one for example, so `defaultBadgeData` may be better off specified explicitly in each respective badge class

Reply:
Will do